lib/bandiera/caching_feature_service.rb

```diff
@@ -4,15 +4,13 @@
 
 module Bandiera
   class CachingFeatureService < SimpleDelegator
-    CACHE_SIZE = 100
-    CACHE_TTL  = 10
 
     attr_reader :cache
 
-    def initialize(delegate)
-      @cache = LruRedux::TTL::ThreadSafeCache.new(CACHE_SIZE, CACHE_TTL)
+    def initialize(delegate, cache_size: DEFAULT_CACHE_SIZE, cache_ttl: DEFAULT_CACHE_TTL)
```

Comment:
lol, this is exactly what I was about to suggest :D

Reply:
Great minds 😂